tests/core/filtering/conftest.py

```diff
@@ -27,7 +27,6 @@ def tester_snapshot(web3):
 
 
 @pytest.fixture(
-    scope='function',
```

Comment:
Although it doesn't really matter, I think you can put this line back just to cut down on lines changed here.